README.markdown

```diff
@@ -1,153 +1,9 @@
 ## Background
-This project is based on the Amazon Cloud Formation framework. It adds a Puppet Face
-to enable simple creation of a fully operational Puppet Enterprise stack utilizing
-module content from the Puppet Forge.
+This project is now deprecated. It was based on the Amazon Cloud Formation framework. 
```

Comment:
This should be AWS CloudFormation. The original text may have been correct at time of publication but is wrong now.